monitor/tasks.py

```
@@ -0,0 +1,489 @@
+        
+        for service in Service.objects.filter(enabled=True):
+            service_results = results.filter(service=service)
+            
+            if not service_results.exists():
+                continue
+                
+            anomalies = _detect_anomalies(service_results)
+            summary = _generate_summary(service, service_results, anomalies)
+            
+            # Create AI digest
+            digest = AIDigest.objects.create(
```

Ruff (0.12.2)

Local variable `digest` is assigned to but never used

Remove assignment to unused variable `digest`

(F841)